spec/beaker_test_helpers.rb

```diff
@@ -0,0 +1,17 @@
+# These are specifically to mock Beaker methods necessary for testing
+# that will be available during runtime because this is never run separate
+# from Beaker itself.
+#
+# Including Beaker as a dependency would not work as a solution to this issue,
+# since that would make a cycle in the dependency graph, at least until
+# Beaker 3.0 happens and this is no longer a dependency of Beaker's.
+module BeakerTestHelpers
+  include Beaker::DSL
+end
+
+module Beaker
```

Comment:
:cool: yep. Seems reasonable.

By doing this in the context of tests you can iterate on the test version of this registration method, so that it collects information and assertions can happen against what it has collected. A lot of times this is identical to a stub or mock, but sometimes by just having a method in a class you can do things like collect callstack info, etc. :shrug: